M0narc/fast_api
# no-story/save-user-in-db

## Description


saved user in db

## Related Story Ticket



## Testing Instructions
create a new user
through the console of endpoint if already created check if the user has been created
through sqlite3 todosapp.db
select * users
you can .quit after checking



## Screenshots


![imagen](https://github.com/M0narc/fast_api/assets/46531321/84d311c3-b64e-45d5-a91c-0fca5938f47d)
